### DISCUSSION TITLE
Same menu items as Plex

### DISCUSSION THREAD

**Discussion by mattm3** [croneter/PlexKodiConnect#1434] [Q&A] [answered]
Hi,

I recently started using PlexKodiConnect and I’m wondering if it is possible to have the same menu items as Plex. For example I have Movies but I also have others videos that aren’t movies and have been setup in Plex as Other Videos but in Kodi they all show under Movies.

Is there a way I can separate them like Plex does or is it a Kodi limitation?

Thanks

**Comment by croneter** [comment, marked as answer]
Yes, you can easily separate your different movie libraries. See here: https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

**Reply by mattm3** [reply]
Thanks croneter, I’ll give this a try. A different skin seems like the way to go. Hopefully there’s one that can replicate Plex.

**Comment by S1llyG1t** [comment]
Sorry to necro this thread but this has been bugging me for a long time. I have used the method to set the video node/movies in my skin(MQ8/Omega). If I don't do this, then all the home movies I have in a separate library on plex get included in movies on kodi. But the small issue I have with this is the skin will no longer display the library information(ie. how many total movies and unwatched movies) like it will for tv shows on the skin menu bar for those sections.

I don't know if this is a PKC or a MQ8 skin issue, or something else I'm not understanding on how to set this up in the skin so I can still see this information, but only see movies in my plex movie library(not home movies).

**Reply by croneter** [reply]
That's a skin issue, dear @S1llyG1t
